packages/addons/browser/chrome/source/default.py

```diff
@@ -74,28 +87,34 @@ def getAudioDevice():
                    })))['result']['value']
 
 def getAudioDeviceType(dev):
-  if dev.startswith("ALSA:"):
-    return "ALSA"
-  if dev.startswith("PULSE:"):
-    return "PULSE"
-  return None
+  if dev:
+    if dev.startswith("ALSA:"):
+      return "ALSA"
+    elif dev.startswith("PULSE:"):
+      return "PULSE"
+    else:
+      # Backward compatibility of device string.
+      return "ALSA"
+  return "ALSA"
 
 def getAlsaAudioDevice(dev):
+  if dev.startswith('plughw:'):
```

Comment:
Here is a similar test required:
```suggestion
  if not dev.startswith('ALSA:'):
```